meta/documents/changelog_en.md

```diff
@@ -5,6 +5,7 @@
 ### Changed
 
 - A change in version 5.0.64 could lead to horizontal scrolling behaviour. This change has been reversed.
+- The settings for ReCaptcha have been revised. If ReCaptcha has been registered as necessary or assigned to the ‘Essential’ group, it will now also be loaded without explicit consent.
```

Comment:
```suggestion
- The settings for reCAPTCHA have been revised. If reCAPTCHA has been registered as necessary or assigned to the ‘Essential’ group, it will be loaded without explicit consent.
```